docs/user-guide/kubectl-overview.md

```diff
@@ -238,6 +238,12 @@ $ kubectl get replicationcontroller <rc-name>
 
 // List all replication controllers and services together in plain-text output format.
 $ kubectl get rc,services
+
+// List all daemon sets, including uninitialized ones, in plain-text output format.
+$ kubectl get ds --include-uninitialized
+
+// List all daemon sets, not including uninitialized ones, in plain-text output format.
+$ kubectl get ds --include-uninitialized=false
```

Comment:
This is not very useful, since it's the default value? Maybe a better example is:

    kubectl get ds NAME --include-uninitialized=false

which would actually do something different when ` --include-uninitialized=false` is present.

Reply:
Does `kubectl get ds NAME --include-uninitialized=false` hide the uninitialized pod? I think if name is provided, the --include-uninitialized=false doesn't take effect.

Agree that line 245, 246 are very useful. Maybe just remove them.

Reply:
"are" or "are not"?

Reply:
“are not". Sorry.